Formula/bore-cli.rb

```diff
@@ -0,0 +1,23 @@
+class BoreCli < Formula
```

Comment:
Let's use `Bore` rather `BoreCli`
```suggestion
class Bore < Formula
```